src/index.js

```diff
@@ -1,6 +1,32 @@
-/**
-* Put your JavaScript here
-*/
+let state = {
+  todos: [
+    { title: 'First todo', complete: false },
+    { title: 'Second todo', complete: true },
+    { title: 'Third todo', complete: true },
+  ],
+};
+
+document.onload = init();
+
+function init() {
+	const listNode = document.getElementById('list');
+
+	const filterButton = document.getElementById('filter-button')
+	filterButton.addEventListener('click', (event) => {
```

Comment:
If you needed to know if the app was filtered or not, how might you be able to tell with this implementation? For example, try to make the filter button say "Hide Completed" if they are not hidden, and "Show Completed" if they are.